app/src/main/java/com/example/newsapp/data/entities/UserDto.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.newsapp.data.entities
+
+data class UserDto(
```

Comment:
Ummm, no need for userDto because it has the same data that in the User class.

If you want to use this pattern maybe rename the User class to UserEntity.

Reply:
done